lib/fizzy/instantiate.cpp

```diff
@@ -408,8 +408,7 @@ std::vector<ExternalFunction> resolve_imported_functions(
                                     " output type doesn't match imported function in module"};
         }
 
-        external_functions.emplace_back(
-            ExternalFunction{std::move(it->function), module_func_type});
+        external_functions.emplace_back(ExternalFunction{it->function, module_func_type});
```

Comment:
Now input `imported_functions` could be passed by const reference, not sure it's worth to change.